che-mount/Dockerfile

```diff
@@ -66,4 +66,6 @@ RUN set -x \
 
 COPY /entrypoint.sh /bin/entrypoint.sh
 
+RUN chmod 744 /bin/entrypoint.sh
```

Comment:
It was not executing when I ran it. Might be the codenvy editor saving it as a non-executable after editing the entrypoint.sh file. It is best just to set this to avoid this problem.